### ISSUE TITLE
kyma apply function errors on validation for "nodejs20" runtime

### ISSUE THREAD

**Issue by vobu** [kyma-project/cli#2071]
**Description**

When trying to `kyma apply function` for (re-)publishing a serverless Node.js function, the validation step fails with `Error: 'nodejs20' invalid runtime`

```shell
$> kyma apply function
X Loading configuration... (4ms)
Error: 'nodejs20' invalid runtime
```

src code excerpt:
```yaml
name: function-sample
namespace: bla
runtime: nodejs20
source:
    sourceType: inline
# ...
```
**Expected result**

validation should pass successfully as `nodejs20` is the default runtime declared via `kyma init function`

**Actual result**

Validation error "Error: 'nodejs20' invalid runtime"

**Steps to reproduce**

0. `kyma init function`
1. (do a nodejs impl)
2. `kyma apply function`

**Comment by halamix2** [comment]
On main branch apply for a nodejs20 function works, but the current release (2.20.3) doesn't recognize nodejs20.

As a workaround before the next release you could
1. use the `nodejs18` runtime
2. `kyma apply function`
3. edit the used runtime in the function with the `kubectl edit function`

**Comment by halamix2** [comment]
Fixed in Kyma 2.20.4
